src/PKSim.Infrastructure/ORM/DAS/DAS.cs

```diff
@@ -153,8 +159,9 @@ public bool Connect(string dbName, string workgroupDb, string user, string passw
             switch (dataProvider)
             {
                case DataProviders.SQLite:
-                  sProvider = "System.Data.SQLite";
+                  sProvider = "Microsoft.Data.Sqlite";
                   connBuilder.Add("Data Source", dbName);
+                  connBuilder.Add("Foreign Keys", "False");
```

Comment:
why false? Shoudn't that be TRUE?

Reply:
I don’t  Think we should change the behaviour of the app just because a test fail.

Reply:
Well I was thinking it's a difference in the default connection parameters but maybe not.

We should find if the test behaviour is different

Reply:
The default connection parameters for `System.Data.SQLite` does not enable foreign key enforcement, so for this PR we can disable it explicitly. @Yuri05 will enter an issue for the template database delete.